frontend/src/styles/globals.scss

```diff
@@ -35,12 +34,23 @@ body,
   // empty space at the bottom of the page, below the footer.
   // Since the user can scroll in `.non-header-wrapper`, everything outside that
   // that still overflows these elements can safely be hidden:
-  overflow: hidden;
+  // overflow: hidden;
 }
 
 html {
   scroll-behavior: smooth;
   scroll-padding-top: $layout-xl;
+  font-family: $font-stack-base;
```

Comment:
Was the font change related to this ticket? Applying this to _everything_ within `<html>` seems like potentially a fairly big change in the sense that I can't quite predict what that will affect?